module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -318,8 +312,7 @@ public function getDefaultMode()
      */
     public function getEdsView()
     {
-        $viewArr = explode('|', $this->getApiProperty('defaultView'));
-        return (1 < count($viewArr)) ? $viewArr[1] : $this->defaultView;
+        return  $this->getDefaultViewPart(1);
```

Comment:
There's an extra space here for some reason. No need for you to take any action, I'm just commenting on it here so I can make the correction through GitHub automatically. ;-)

```suggestion
        return $this->getDefaultViewPart(1);
```